sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/DocumentAnalysisLiveTestBase.cs

```diff
@@ -130,8 +128,25 @@ protected async Task<DisposableDocumentModel> BuildDisposableDocumentModelAsync(
                 _ => TestEnvironment.BlobContainerSasUrl,
             };
             var trainingFilesUri = new Uri(trainingFiles);
+            var buildMode = DocumentBuildMode.Template;
+            var modelId = Recording.GenerateId();
+
+            skipCaching |= (Recording.Mode == RecordedTestMode.Record) || (Recording.Mode == RecordedTestMode.Playback);
```

Comment:
With caching, we may end up skipping some service calls, which can affect record/playback. For this reason, we're disabling it for these modes.